go/host/interfaces.go

```diff
@@ -4,17 +4,42 @@ import (
 	gethcommon "github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/obscuronet/go-obscuro/go/common"
+	"github.com/obscuronet/go-obscuro/go/config"
+	"github.com/obscuronet/go-obscuro/go/ethadapter"
+	"github.com/obscuronet/go-obscuro/go/host/db"
 )
 
-// P2PCallback -the glue between the P2p layer and the node. Notifies the node when rollups and transactions are received from peers
-type P2PCallback interface {
+// Host is the half of the Obscuro node that lives outside the enclave.
+type Host interface {
```

Comment:
chatted on discord. It woudl be good to break this up in 2 interfaces: "Host" and "MockHost"

Reply:
Done.